### PR TITLE
data: adds a new entry for Anomaly Detection

### PR DESCRIPTION
**Pull request by lanterno**

### CODE REVIEW

File: data/MLO/projects.yaml [hunk 1]
```diff
@@ -490,3 +490,19 @@ projects:
...
     date_added: 2022-07-07
     date_updated: 2022-07-07
+
+  anomaly-detection:
+    name: Distributed Homomorphic Anomaly Detection
+    categories:
+      - Learning
+    applications:
+      - Info
+    description: Runs anomaly detection algorithms in a distributed and encrypted fashion
```
Comment (ineiti) [inline data/MLO/projects.yaml:500]: The description should be at least 1 paragraph, better 2 paragraphs. It should describe:
- what problem it solves
- how it solves it
- why it's better than other solutions

**Review by ineiti — CHANGES_REQUESTED** [review]
I think we should also link to the repo you used for the project - or is there any sensitive information inside?

**Comment by lanterno** [comment]
@ineiti I don't think there's anything sensitive in the codebase. The repo is currently private, should I go ahead and make it public? 
Here's the link to the code to check first: https://github.com/c4dt/predictive-maintenance
The only thing that mentions RUAG is the short readme file.

**Comment by ineiti** [comment]
Great - go ahead with making it public then, and link it in the showcase entry.

File: data/MLO/projects.yaml [hunk 2]
```diff
@@ -490,3 +490,30 @@ projects:
...
     date_added: 2022-07-07
     date_updated: 2022-07-07
+
+  anomaly-detection:
+    name: Distributed Homomorphic Anomaly Detection
+    categories:
+      - Learning
+    applications:
+      - Info
+    description: Runs anomaly detection algorithms in a distributed and encrypted fashion.
```
Comment (ineiti) [inline data/MLO/projects.yaml:500]: As we're not doing any encryption here, the description should reflect that.
```suggestion
    description: Proof-of-concept ML algorithms to do anomaly detection compatible with distributed, encrypted algorithms.
```

**Review by ineiti — CHANGES_REQUESTED** [review]

File: data/MLO/projects.yaml (hunk 2 above)
Comment (lanterno) [reply data/MLO/projects.yaml]: Makes sense. added your suggestion.

**Review by ineiti — APPROVED** [review]

**Pull request merged**